src/main/java/ru/neoflex/chmutenko/bank/CreditConveyor/controller/ApplicationController.java

```diff
@@ -0,0 +1,51 @@
+package ru.neoflex.chmutenko.bank.CreditConveyor.controller;
+
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.annotation.*;
+import ru.neoflex.chmutenko.bank.CreditConveyor.dto.LoanRequestDTO;
+import ru.neoflex.chmutenko.bank.CreditConveyor.dto.LoanOfferDTO;
+import ru.neoflex.chmutenko.bank.CreditConveyor.exceptions.DataNotValidException;
+import ru.neoflex.chmutenko.bank.CreditConveyor.service.ApplicationService;
+
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@RestController
+@RequestMapping("/conveyor/offers")
+public class ApplicationController {
+
+    private final ApplicationService applicationService;
+    private static final Logger logger = LoggerFactory.getLogger(ApplicationController.class);
+
+    @Autowired
+    public ApplicationController(ApplicationService applicationService) {
+        this.applicationService = applicationService;
+    }
+
+    @PostMapping()
+    public List<LoanOfferDTO> offerDTOs(@RequestBody @Valid LoanRequestDTO loanRequestDTO,
```

Comment:
```suggestion
    public ResponseEntity<List<LoanOfferDTO>> offerDTOs(@RequestBody @Valid LoanRequestDTO loanRequestDTO,
```
мы оборачиваем все ответы в Response Entity

Reply:
Есть подход, когда ReponseEntity не используется, если сможешь обосновать, разрешу оставить как есть!